nautobot/core/templates/inc/custom_fields/panel_data.html

```diff
@@ -3,37 +3,58 @@
         </div>
-        <table class="table table-hover panel-body attr-table">
-            {% for field, value in custom_fields.items %}
-                <tr>
-                    <td><span title="{{ field.description }}">{{ field }}</span></td>
-                    <td>
-                        {% if field.type == 'boolean' %}
-                            {{ value | render_boolean }}
-                        {% elif field.type == 'url' and value %}
-                            <a href="{{ value }}">{{ value|truncatechars:70 }}</a>
-                        {% elif field.type == 'multi-select' and value %}
-                            {{ value|join:", " }}
-                        {% elif field.type == 'json' and value is not None %}
-                            <p>
-                                <button class="btn btn-xs btn-primary" type="button" data-toggle="collapse"
-                                        data-target="#cf_{{ field.name | slugify }}" aria-expanded="false"
-                                        aria-controls="cf_{{ field.name | slugify }}">
-                                    Show/Hide
-                                </button>
-                            </p>
-                            <pre class="collapse" id="cf_{{ field.name | slugify }}"><code>{{ value|render_json }}</code></pre>
-                        {% elif value or value == 0 %}
-                            {{ value }}
-                        {% elif field.required %}
-                            <span class="text-warning">Not defined</span>
-                        {% else %}
-                            <span class="text-muted">&mdash;</span>
-                        {% endif %}
-                    </td>
-                </tr>
-            {% endfor %}
+        <table id="accordion" class="table table-hover panel-body attr-table">
+            {% for grouping, fields in custom_fields.items %}
+            {% with forloop.counter0 as count %}
+                {% if grouping != "" %}
+                    <tr>
+                        <!-- #TODO
+                        Here we are using forloop.counter0 to guarantee the unqiueness of `data-target` attribute.
+                        This feature on the joblist page is using {{ grouping | slugify }} it should be changed as it breaks `accordion-toggle`
+                        when two distinct groups have the same slugified name. e.g. Group 2, group-2 -->
+                        <th>
+                            <button type="button" class="btn-link accordion-toggle mdi mdi-chevron-down"
+                            name="grouping.{{ grouping }}" data-toggle="collapse"
+                            data-target=".collapseme-{{ count }}">
+                                {{ grouping }}
+                            </button>
+                        </th>
+                        <td></td>
```

Comment:
Perhaps would be better to use a `colspan="2"` on the first th/td so that it just extends to the full width of the table?